### PR TITLE
Modify our-projects selection

### PR DESCRIPTION
**Pull request by Polo2**
Update selection of projects:
- Remove Skiwallet :disappointed_relieved: 
- Add Iperia, Accorlocal, Sirac

### CODE REVIEW

File: locales/fr.yml [hunk 1]
```diff
@@ -24,11 +24,26 @@ fr:
       main-title: Nous sommes développeurs <span>Ruby/Rails</span>
       subtitle: experts en backend, MVPs &amp; open-source
     our-projects:
+      accorlocal:
+        description: Réservation de services de proximité dans les hôtels du groupe Accor. Développement back-end et back-office.
+        link: https://www.accorlocal.com/
+        stack: Rails / API Restfull / Rspec / PostgreSQL
```
Comment (michaelbaudino) [inline locales/fr.yml]: `RESTful`, not `Restfull`... and `RSpec`, not `Rspec`... but I think we should even change it to:

```suggestion
        stack: Rails / API (Grape) / ActiveAdmin / RSpec / PostgreSQL
```

File: locales/fr.yml [hunk 2]
```diff
@@ -24,11 +24,26 @@ fr:
...
+        title: AccorLocal
+      goodsh:
+        description: Application sociale de partage de bons plans. Développement back-end et API.
+        link: https://goodsh.it/
+        stack: Rails / API / Trailblazer / Rspec / Redis
```
Comment (michaelbaudino) [inline locales/fr.yml]:
```suggestion
        stack: Rails / API / Trailblazer / RSpec / Redis
```

File: locales/fr.yml [hunk 3]
```diff
@@ -24,11 +24,26 @@ fr:
...
+        title: Goodsh
       guerir-le-regard:
         description: Web-documentaire pour l'Institut Curie. Prix du Jury du Web Program Festival.
         link: http://guerirleregard.fr
         stack: Middleman / Sass / CoffeeScript
         title: Guérir le regard
+      iperia:
+        description: CVthèque en ligne pour les salariés du domicile.
+        link: https://moncv.iperia.eu/
+        stack: Rails / API / Rspec / Coffeescript / Saas
```
Comment (michaelbaudino) [inline locales/fr.yml]:
```suggestion
        stack: Rails / API / RSpec / CoffeeScript / Sass
```

File: locales/en.yml [hunk 4]
```diff
@@ -43,18 +58,18 @@ en:
         description: Gay-friendly Airbnb. High growth website. Short iterations. Large feature set.
         link: https://www.misterbandb.com
         stack: Rails / PostgreSQL / Rspec
-        title: misterb&b
-      misterout:
+        title: Misterb&b
```
Comment (michaelbaudino) [inline locales/en.yml]: No, I think there's no uppercase: https://www.misterbandb.com/pages/join-us

```suggestion
        title: misterb&b
```

File: locales/en.yml [hunk 5]
```diff
@@ -43,18 +58,18 @@ en:
...
+      sirac:
+        description: Provision of part-time staff. Back-end and front-end development.
+        link: http://www.sirac-ettp-temps-partiel.fr/
+        stack: Rails / Rspec / PostgreSQL
```
Comment (michaelbaudino) [inline locales/en.yml]:
```suggestion
        stack: Rails / RSpec / PostgreSQL
```

File: locales/en.yml [hunk 6]
```diff
@@ -24,11 +24,26 @@ en:
       main-title: We are <span>Ruby/Rails</span> developers
       subtitle: experts in backend, MVPs & open source
     our-projects:
+      accorlocal:
+        description: description
+        link: https://www.accorlocal.com/
+        stack: Rails / API Restfull / Rspec / PostgreSQL
```
Comment (michaelbaudino) [inline locales/en.yml]:
```suggestion
        stack: Rails / API (Grape) / RSpec / PostgreSQL
```

File: locales/en.yml [hunk 7]
```diff
@@ -24,11 +24,26 @@ en:
...
+        title: AccorLocal
+      goodsh:
+        description: Social application of sharing good tips. Back-end and API development.
+        link: https://goodsh.it/
+        stack: Rails / API / Trailblazer / Rspec / Redis
```
Comment (michaelbaudino) [inline locales/en.yml]:
```suggestion
        stack: Rails / API / Trailblazer / RSpec / Redis
```

File: locales/en.yml [hunk 8]
```diff
@@ -24,11 +24,26 @@ en:
...
+        title: Goodsh
       guerir-le-regard:
         description: Web-documentary for Institut Curie. Jury prize at Web Program Festival.
         link: http://guerirleregard.fr
         stack: Middleman / Sass / CoffeeScript
         title: Guérir le regard
+      iperia:
+        description: CV database for home-based employees.
```
Comment (michaelbaudino) [inline locales/en.yml]:
```suggestion
        description: Resumes database for domestic employees.
```

File: locales/en.yml [hunk 9]
```diff
@@ -43,18 +58,18 @@ en:
...
         stack: Rails / PostgreSQL / Rspec
-        title: misterb&b
-      misterout:
+        title: Misterb&b
+      sirac:
+        description: Provision of part-time staff. Back-end and front-end development.
+        link: http://www.sirac-ettp-temps-partiel.fr/
+        stack: Rails / Rspec / PostgreSQL
+        title: Sirac
+      title: Some projects
```
Comment (michaelbaudino) [inline locales/en.yml]: Is this supposed to be here? :thinking:

File: locales/fr.yml [hunk 10]
```diff
@@ -24,11 +24,26 @@ fr:
       main-title: Nous sommes développeurs <span>Ruby/Rails</span>
       subtitle: experts en backend, MVPs &amp; open-source
     our-projects:
+      accorlocal:
+        description: Réservation de services de proximité dans les hôtels du groupe Accor. Développement back-end et back-office.
```
Comment (michaelbaudino) [inline locales/fr.yml]:
```suggestion
        description: Réservation de services de proximité dans les hôtels du groupe Accor. Développement back-end, API et back-office.
```

File: locales/fr.yml [hunk 11]
```diff
@@ -43,18 +58,18 @@ fr:
         description: Airbnb gay-friendly. Site en très forte croissance. Itérations courtes. Nombreuses fonctionnalités.
         link: https://www.misterbandb.com
         stack: Rails / PostgreSQL / Rspec
-        title: misterb&b
-      misterout:
+        title: Misterb&b
+      sirac:
+        description: Mise à disposition de salariés à temps partiel / partagé. Développement de l'application web complète.
```
Comment (michaelbaudino) [inline locales/fr.yml]:
```suggestion
        description: Mise à disposition de salariés à temps partiel / partagé. Outil complet du coeur de métier : gestion des RH. Développement fullstack.
```

File: locales/fr.yml [hunk 12]
```diff
@@ -43,18 +58,18 @@ fr:
...
+        link: http://www.sirac-ettp-temps-partiel.fr/
+        stack: Rails / Rspec / PostgreSQL
```
Comment (michaelbaudino) [inline locales/fr.yml]:
```suggestion
        stack: Rails / RSpec / PostgreSQL / SemanticUI
```

File: locales/fr.yml [hunk 13]
```diff
@@ -43,18 +58,18 @@ fr:
...
+        title: Sirac
+      title: Quelques projets
```
Comment (michaelbaudino) [inline locales/fr.yml]: Is this supposed to be here? :thinking:

**Comment by michaelbaudino** [comment]
Hrm, we should also probably add [Save.co](https://save.co)... We crafted their MVP/prototype during (before?) their first skyrocket-growth rally :thinking:

File: locales/fr.yml [hunk 14]
```diff
@@ -44,17 +59,22 @@ fr:
         link: https://www.misterbandb.com
         stack: Rails / PostgreSQL / Rspec
         title: misterb&b
-      misterout:
+      save:
+        description: Service de réparation d'appareils numériques.
+        link: https://www.save.co/fr/fr/
+        stack: "???"
```
Comment (thibaultdalban) [inline locales/fr.yml]: Rails / RSpec / CoffeeScript / Sass/ PostgreSQL

File: locales/fr.yml [hunk 15]
```diff
@@ -44,17 +59,22 @@ fr:
         link: https://www.misterbandb.com
         stack: Rails / PostgreSQL / Rspec
         title: misterb&b
-      misterout:
+      save:
+        description: Service de réparation d'appareils numériques.
+        link: https://www.save.co/fr/fr/
```
Comment (thibaultdalban) [inline locales/fr.yml]: `https://www.save.co/`

File: locales/en.yml [hunk 16]
```diff
@@ -44,17 +59,22 @@ en:
...
         stack: Rails / PostgreSQL / Rspec
         title: misterb&b
-      misterout:
+      save:
+        description: Repair service for electronic devices.
+        link: https://www.save.co/
+        stack: Rails / RSpec / CoffeeScript / Sass/ PostgreSQL
+        title: Save
+      sirac:
+        description: Provision of part-time staff. Fullstack development.
```
Comment (michaelbaudino) [inline locales/en.yml]:
```suggestion
        description: Provision of part-time staff. Fullstack development of post-MVP prototype in high-growth phase.
```

File: locales/fr.yml [hunk 17]
```diff
@@ -44,17 +59,22 @@ fr:
         link: https://www.misterbandb.com
         stack: Rails / PostgreSQL / Rspec
         title: misterb&b
-      misterout:
+      save:
+        description: Service de réparation d'appareils numériques.
```
Comment (michaelbaudino) [inline locales/fr.yml]:
```suggestion
        description: Service de réparation d'appareils numériques. Développement fullstack du prototype post-MVP supportant une forte croissance.
```

**Pull request merged**